src/client/java/minicraft/level/tile/GrassTile.java

```diff
@@ -66,7 +70,7 @@ public boolean interact(Level level, int xt, int yt, Player player, Item item, D
 			if (tool.type == ToolType.Hoe) {
 				if (player.payStamina(4 - tool.level) && tool.payDurability()) {
 					int data = level.getData(xt, yt);
-					level.setTile(xt, yt, Tiles.get("Dirt"));
+					level.setTile(xt, yt, Tiles.get("Farmland"));
```

Comment:
Revert this.

Reply:
Hoeing a grass tile should also making it into a farmland. Would that not be redundant? Shovels should only be used for removing grasses on the grass tiles.

Reply:
Okay, it's good as long as it still has a chance to drop seeds.